Framework/PythonInterface/plugins/algorithms/LoadVesuvio.py

```diff
@@ -142,13 +142,46 @@ def PyExec(self):
     def validateInputs(self):
         issues = {}
 
-        # Validtae run number ranges
+        # Validate run number ranges
         run_str = self.getProperty(RUN_PROP).value
         if "-" in run_str:
             lower, upper = run_str.split("-")
             if upper < lower:
                 issues[RUN_PROP] = "Range must be in format lower-upper"
 
+
+        # Validate SpectrumList
+        speclists_str = self.getProperty(SPECTRA_PROP).value
+
+        if ";" in speclists_str:
+            # Split if in 2-3;6-7 format
+            speclists_str = speclists_str.split(";")
+        else:
+            # Treat spec as a list
+            speclists_str = [speclists_str]
+
+        for speclist_str in speclists_str:
+            if "-" in speclist_str:
+                lower, upper = speclist_str.split("-")
+                lower = int(lower)
+                upper = int(upper)
+                # Check format
+                if upper < lower:
+                    issues[SPECTRA_PROP] = "Range must be in format lower-upper"
+                # Check Min/Max boundaries
+                if lower < 3:
```

Comment:
Can we avoid the duplication of `3` &  `198`? The instrument parameter file already contains the information about the range of spectrum numbers for the forward- and backward- scattering banks. It would be nice not to have the information duplicated in several places.

There is a method `def _load_common_inst_parameters()` that attaches properties to the object. Amongst these are `self._backward_spectra_list` & `self._forward_spectra_list`. The first and last values of these lists respectively could be checked?